activestorage/config/routes.rb

```diff
@@ -23,11 +25,32 @@
   resolve("ActiveStorage::VariantWithRecord") { |variant, options| route_for(:rails_representation, variant, options) }
   resolve("ActiveStorage::Preview") { |preview, options| route_for(:rails_representation, preview, options) }
 
-
   direct :rails_blob do |blob, options|
     route_for(:rails_service_blob, blob.signed_id, blob.filename, options)
   end
 
   resolve("ActiveStorage::Blob")       { |blob, options| route_for(:rails_blob, blob, options) }
   resolve("ActiveStorage::Attachment") { |attachment, options| route_for(:rails_blob, attachment.blob, options) }
+
+  direct :rails_storage do |model, options|
+    if model.respond_to?(:signed_id)
+      route_for(
+        options[:deliver_by] == :proxy ? :rails_service_blob_proxy : :rails_service_blob,
```

Comment:
Curious if we want `active_storage_proxy_path` instead?

Reply:
Hm, well, all the other routes are scoped with `rails_`, so all good 👍

Reply:
I'm almost tempted to drop the rails_ prefix for this, though. Because they're meant to be used directly. Like it was suggested with asset_path.

Reply:
Sure, we've already shipped `rails_` prefixed URLs so in this case, breaking the consistency doesn't provide too much of a win. Agree on the almost temptation 😄

Reply:
@dhh out of interest, why do you dislike the deliver_by options?

personally, I prefer learning fewer method names, and using options to specialise like with 

`asset_path("application", type: :javascript, skip_pipeline: true)`

or for that matter `link_to ... remote:true`

Reply:
@kaspeth I think rails_ extensions make sense for all the URLs you're not commonly likely to set yourself. But the temptation is much larger if you're expected to call these with regularity. Anyway, probably best not to do it.

@confusedvorlon Because these are different routes. Having a single route that takes an option to give you different routes doesn't make sense to me. It's not even shorter!